spec/models/user_spec.rb

```diff
@@ -1,4 +1,133 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
+  subject { described_class.new }
+
+  it 'user is valid' do
+    subject.first_name = 'John'
+    subject.last_name = 'Doe'
+    subject.email = 'johndoe@gmail.com'
+    subject.username = 'jdoe'
+    subject.phone_number = '0449 774 759'
+
+    expect(subject).to be_valid
+  end
```

Comment:
This is super verbose. We can use the `shoulda-matchers` to make this more concise:
```ruby
subject { build :user }

it { is_expected.to be_valid }
it { is_expected.to validate_presence_of :first_name }
it { is_expected.to validate_presence_of :username }
it { is_expected.to validate_uniqueness_of(:username).case_insensitive }
```

(style) if you're referencing `subject` explicitly, name it:
```ruby
subject(:user) { described_class.new }

it { expect(user).to be_valid }
```

Reply:
Ah brilliant okay! I wasn't aware of the factories as well to simplify this; so I'll rework this file with the recommendations, thanks Matt!